advanced_functionality/autogluon-tabular/container-training/train.py

```diff
@@ -19,6 +19,11 @@
 import shutil
 import networkx as nx
 
+import shap
+import smdebug.mxnet as smd
+from sagemaker.debugger import DebuggerHookConfig
```

Comment:
Remove `from sagemaker.debugger import DebuggerHookConfig`.
This class is not used in the training script.

Reply:
Thanks, have removed the line.